test/system/use_worker_pool_tests.rb

```diff
@@ -61,29 +61,26 @@ def work!(seconds); sleep(seconds); end
     end
 
     should "spawn and wait until work is available" do
-      # the workers should be spawned and waiting
-      assert_equal 2,    subject.waiting
+      # the workers should be spawned and available
       assert_equal true, subject.worker_available?
 
-      # only one worker should be waiting
+      # only one worker should be available
       subject.add_work 5
-      assert_equal 1,    subject.waiting
```

Comment:
@jcredding case-in-point on the "available worker count" method: I don't love that you state only one worker should be available but don't prove it with an assertion.